src/app/data-explorer/querybuilder/querybuilder.component.ts

```diff
@@ -52,6 +52,11 @@ export class QueryBuilderComponent implements OnInit {
   operatorMap: { [key: string]: string[] } = {
     /* eslint-disable-next-line */
     string: ['=', '!=', 'contains', 'like', 'startswith', 'endswith'],
+    datetime: ['=', '!=', '>', '<', '=>', '=<'],
+    date: ['=', '!=', '>', '<', '=>', '=<'],
+    time: ['=', '>', '<', '=>', '=<'],
+    boolean: ['=', '!='],
+    number: ['=', '!=', '>', '<', '=>', '=<'],
```

Comment:
The Query Builder has default operators for it's default types. Therefore, it is only necessary to add operators here when the defaults are to be overridden or the type is not a default Query Builder type. This means that line 55 (datetime) should be retained and lines 56-59 (date, time, boolean and number) can be deleted since they are not required.

Please also add a comment to the code explaining this to avoid confusion in the future.